library/kotlin/src/io/envoyproxy/envoymobile/RetryPolicy.kt

```diff
@@ -8,19 +8,19 @@ package io.envoyproxy.envoymobile
  * @param perRetryTimeoutMs Timeout (in milliseconds) to apply to each retry.
  */
 data class RetryPolicy(
-    val maxRetryCount: Int,
-    val retryOn: List<RetryRule>,
-    val perRetryTimeoutMs: Long?
+  val maxRetryCount: Int,
+  val retryOn: List<RetryRule>,
+  val perRetryTimeoutMs: Long? = null
```

Comment:
@rebello95 How do you feel about this default to null?

Reply:
Seems fine